### ISSUE TITLE
Cannot add action with URL /rules/<id> to rule

### ISSUE THREAD

**Issue by Thomas-Vos** [dresden-elektronik/deconz-rest-plugin#1174]
I'm not able to add the following action to a rule:
```
{
  "address": "/rules/1",
  "body": {
    "status": "disabled"
  },
  "method": "PUT"
}
```
deCONZ answers with the following response:
```
[
  {
    "error": {
      "address": "/rules/1",
      "description": "Rule actions contain errors or an action on a unsupported resource",
      "type": 608
    }
  }
]
```
I think this is because an action with URL `/rules/<id>` is not allowed on a rule. This is allowed on the Hue bridge. I need this so I can disable a rule when a rule is activated. Is this something that could be supported?

**Comment by manup** [comment]
Good catch, it is fixed now and will be part of version 2.05.58.

**Comment by Thomas-Vos** [comment]
Thanks! Just tested this and it's working now.
